### PR TITLE
Blog post about 1.14 release.

### LINKED ISSUE #235: Blog post about Kubewarden 1.14 release
Write a blog post covering these topics:

- new host capability
- kwctl scaffold vap
- CEL policy: mention artifacthub + the existing blog post about it
- finalizers
- KCD italy?
-  Known bugs: mention policy-server cert expiration of 1 year
- policy fixes
   - https://github.com/kubewarden/environment-variable-policy/issues/13
   - https://github.com/kubewarden/container-resources-policy/issues/33

### PR DESCRIPTION
**Pull request by jvanz**
## Description

Adds a blog post to show what was accomplished in the v1.14 release.

Fix #235

### CODE REVIEW

File: content/blog/2024/06/kubewarden-1-14-release.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+1. **[Environment Variable
+   Policy](https://github.com/kubewarden/environment-variable-policy/issues/13)**:
+   Previously, variables with values were not blocked if only the name was
+   specified in the settings. We have updated the validation code to ensure
+   that the policy will validate the names even when the user does not define
+   the envvar value in the rule.
+   
+2. **[Container Resources
+   Policy](https://github.com/kubewarden/container-resources-policy/issues/33)**:
```
Comment (flavio) [inline content/blog/2024/06/kubewarden-1-14-release.md:65]:
```suggestion
   Policy](https://artifacthub.io/packages/kubewarden/container-resources/container-resources)**:
```

File: content/blog/2024/06/kubewarden-1-14-release.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   Users faced issues when trying to create policies with only CPU settings
+   empty memory values. The policy settings were sent with empty values,
+   causing validation errors. We have fixed this by allowing a single resource
+   setting to be empty, resolving the issue without breaking policy usage.
```
Comment (flavio) [inline content/blog/2024/06/kubewarden-1-14-release.md:69]:
```suggestion
   setting to be empty, resolving the issue without breaking policy usage. The problem was describe in depth by [this issue](https://github.com/kubewarden/container-resources-policy/issues/33).
```

File: content/blog/2024/06/kubewarden-1-14-release.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+The Kubewarden controller now follows best practices by using domain-qualified
+finalizers. Users must be aware that if they downgrade a Kubewarden
+installation, the old controller will not be able to identify the new
+finalizer. Leaving the cluster in a state where the resource could not be
+removed requiring manual intervention. 
+
+NOTE: Downgrade procedures are not supported by Kubewarden project. See the
+[documentation](https://docs.kubewarden.io/reference/upgrade-path#downgrades)
+to learn more about that
```
Comment (flavio) [inline content/blog/2024/06/kubewarden-1-14-release.md:81]:
```suggestion
to learn more about that.
```

File: content/blog/2024/06/kubewarden-1-14-release.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+## KCD Italy: CEL for Cluster Security
+
+One of the Kubewarden maintainers,  [Flavio
+Castelli](https://github.com/flavio/),  delivered an insightful talk at [KCD
+Italy](https://community.cncf.io/events/details/cncf-kcd-italy-presents-kcd-italy-2024/)
+on "How to leverage and extend CEL for your cluster security." He explored the
+capabilities and trade-offs of the CEL-based admission controller introduced in
+Kubernetes 1.28 and discussed how to use CEL with other dynamic admission
+controllers. This talk highlighted the practical applications and advantages of
+using CEL for cluster security.
```
Comment (flavio) [inline content/blog/2024/06/kubewarden-1-14-release.md:92]:
```suggestion
using CEL for cluster security.

While the talk has been delivered in Italian, the slides are written in English and can be found [here](http://localhost:1313/2024/06/25/kcd-italy---cel-kubernetes-validatingadmissionpolicy-and-kubewarden/).
```

Note: I'm currently experiencing some DNS issue with my personal website, the link is broken right now but it should be working once the DNS changes are propagated :crossed_fingers:

File: content/blog/2024/06/kubewarden-1-14-release.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+## Known Bugs and Future Plans
+
+During the release process, we identified a bug related to the policy server
+certificate rotation. While the Root CA is set to expire in 10 years, each
+policy-server cert secret has a one-year expiry. But the controller is
+currently unable to renew them automatically. In this release, we will ensure
+that policy-server secrets are created with a 10-year expiry. And in future
+releases we plan to properly implement an automated renewal process. For now,
+users can manually delete the expired cert secret (`policy-server-default`) and
+trigger the controller reconciliation by adjusting the replicas.
```
Comment (flavio) [inline content/blog/2024/06/kubewarden-1-14-release.md:103]:
```suggestion
trigger the controller reconciliation by adding/removing/updating a policy or by adjusting the number of replicas of a `PolicyServer`.
```

We are going to work on a better solution to this problem in an upcoming release of Kubewarden.

**Review by flavio — CHANGES_REQUESTED** [review]
Thanks a lot for having handled that

File: content/blog/2024/06/kubewarden-1-14-release.md (hunk 4 above)
Comment (viccuad) [reply content/blog/2024/06/kubewarden-1-14-release.md:92]: Edited Flavio's suggestion with the correct link, previously it was `localhost:1313`.

**Review by viccuad — APPROVED** [review]
LGTM modulo Flavio's suggestions.

**Review by flavio — APPROVED** [review]

**Comment by flavio** [comment]
Just remember to change the article's date please

**Review by viccuad — APPROVED** [review]

**Pull request merged**